mllib-dal/src/main/native/ALSDALImpl.cpp

```diff
@@ -212,7 +214,7 @@ void initializeStep2Local(
 
 void initializeModel(size_t rankId, ccl::communicator &comm, size_t partitionId,
                      size_t nBlocks, size_t nUsers, size_t nFactors) {
-    std::cout << "ALS (native): initializeModel " << std::endl;
+    logger::print(logger::INFO, "ALS (native): initializeModel \n");
```

Comment:
I think you can provide two functions: print and println then you don't need to add "\n" in every call.